doc/source/examples/multiphysics/tracer-through-cad-junction/tracer-through-cad-junction.rst

```diff
@@ -0,0 +1,161 @@
+
+Features
+----------------------------------
+- Solver: ``gls_navier_stokes_3d`` 
+- Transient problem
+- Displays the use of the tracer physics
+- Displays the use of a simplex mesh generated with a CAD platform
+
+Location of the example
+------------------------
+``examples/multiphysics/tracer-through-cad-junction/tracer_through_cad_junction.prm``
+
+Description of the case
+-----------------------
+
+A junction of pipes following an arbitrary path, in this case orthogonal sinusoids, is difficult to mesh properly with hexaedral elements. 
+A simplex mesh, on the other hand, is easy to setup and can be done with many well-established algorithms; all software that can create a mesh can create a simplex mesh.
+This case shows how a passive tracer injected in such a geometry can be used to visualize preferential paths.
+
+
+Generating a geometry with SALOME
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Complex geometries can be set up and meshed with the SALOME platform. 
+
+First, the geometry module must be loaded as such, and geometric shapes can be added. For example, here, disks are drawn alongside sinusoidal curves that will be used for pipe generation through extrusion. The Fuse command can then be used to obtain a single geometry.
+
+.. image:: images/salome_menu.png
+    :alt: Salome menu with geometry visible
+    :align: center
+
+.. image:: images/salome_geometry.png
+    :alt: Geometry generated with Salome
+    :align: center
+
+Second, the mesh module can be loaded to generate a ``.mesh`` file. A new mesh must be created with an associated geometry, algorithm and hypothesis. NETGEN 1D-2D-3D works well, and automatic hypothesis selection can be used. The element sizes can be edited to specific requirements, and then the Compute command must be used to generate the grid. Through File->Export->GMF File, the ``.mesh`` file can be output.
+
+.. image:: images/salome_mesh_creation.png
+    :alt: Contextual menu for mesh generation in Salome
+    :align: center
+
+
+Adding the boundary IDs from Gmsh
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A criterion for the use of ``.msh`` file in Lethe is that the boundaries have an associated ID number. These numbers can be added in Gmsh.
+
+The boundaries from the mesh must be linked to a Physical object. In 3D for example, the inlet, outlet and walls must be specified as Physical surfaces. Selecting the surface for the inlet and outlet can be done easily, and then the ``.geo`` file can be text-edited to add a new physical surface with the remaining surfaces. The physical volume must also be specified.
+In this specific case, there are four boundaries: inlets with no tracer (1), inlet with tracer (2), walls (3) and outlets (4).
+
+.. code-block:: text
+
+    Merge "Mesh_1.mesh";
+    Physical Surface(1) = {729, 712};
+    Physical Surface(2) = {10};
+    Physical Surface(3) = {1:9,11:711,713:728,730:848,850,852:5000};
+    Physical Surface(4) = {849, 851};
+    Physical Volume(1) = {1};
```

Comment:
```suggestion

The ``Physical Volume`` must also be specified for compatibility with deal.II, as stated with this line from the ``.geo`` file:

.. code-block:: text

    Physical Volume(1) = {1};

```

I think it will be clearer this way.

Reply:
Good suggestion!